examples/amp-story/amp-story-audio-sticker.html

```diff
@@ -0,0 +1,200 @@
+    }
+    amp-story-page {
+      background: white;
+    }
+    amp-story h1 {
+      font-size: 2.875rem;
+    }
+    amp-story h2 {
+      font-size: 2.25rem;
+    }
+    amp-story p {
+      line-height: 1.5;
+    }
+    .bold {
+      font-weight: bold;
+    }
+    .bottom {
+      align-content: end;
+    }
+    .medium {
+      font-weight: 600;
+    }
+    .first {
+      padding-top: 8.83vh;
+    }
+    .last {
+      padding-bottom: 8.83vh;
+    }
+    .blue {
+      color: #4285F4;
+    }
+    .twenty-px {
+      font-size: 1.25rem;
+    }
+    .center {
+      text-align: center;
+    }
+    .icon {
+      background-image: url('./img/AMP-Brand-White-Icon.svg');
+      background-repeat: no-repeat;
+      background-size: 4.25rem 4.25rem;
+      height: 4.25rem;
+      object-fit: contain;
+      width: 4.25rem;
+    }
+    .sticker {
+      height: 3rem;
+      width: 3rem;
+    }
+    .container {
+      position: relative;
+      width: 180px;
+      height: 180px;
+      overflow: hidden;
+    }
+    .byline {
+      letter-spacing: 1.28px;
+      padding-bottom: 7.88vh;
+    }
+    .introducing p {
+      line-height: 2.625em;
+    }
+    .introducing h2 {
+      line-height: 1.17em;
+    }
+    .subtitle-page {
+      padding-top: 10.87vh;
+    }
+    .button {
+      align-items: center;
+      border: 4px solid #FFFFFF;
+      color: #FFFFFF;
+      display: flex;
+      margin: 0 auto;
+      max-width: 240px;
+      text-decoration:none;
+    }
+    .button p {
+      width: 100%;
+    }
+    amp-ad[template="image-template"] img {
+      object-fit: cover;
+    }
+
+    ::cue {
+      background-color: rgba(0, 0, 0, 0.75);
+      font-size: 24px;
+      line-height: 1.5;
+    }
+  </style>
+</head>
+  <body>
+    <amp-story standalone id="cats"
+        title="Key Highlights of AMP Conf 2018" publisher="The AMP team"
+        publisher-logo-src="./img/amplogo.png"
+        poster-portrait-src="./img/overview.jpg">
+
+      <amp-story-page id="page-1" title="Lorem ipsum dolor sit amet">
+        <amp-story-grid-layer template="fill">
+          <amp-video autoplay loop
+            width="400"
+            height="750"
+            poster="./img/poster0.png"
+            layout="fill">
+            <source src="./video/p1.mp4" type="video/mp4">
+          </amp-video>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical" class="bottom">
+          <div class="icon"></div>
+          <h1 class="bold">Key Highlights of AMP Conf 2018</h1>
+          <p class="byline">By The AMP team</p>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer>
+          <amp-story-audio-sticker class="sticker">
+            <amp-img width="100" height="100" layout="responsive" src="https://media.tenor.com/oK5AJr1cvF4AAAAi/high-volume-speaker-symbols.gif">
+          </amp-story-audio-sticker>
+        </amp-story-grid-layer>>
+      </amp-story-page>
+
+      <amp-story-page id="page-2">
```

Comment:
This demo could only have 2 pages to show how it works (both pages with an audio sticker to show that both disappear when unmuting any page). The other pages are not that useful.

Reply:
Updated.